public/v4/apps/fireflyiii-data-importer.yml

```diff
@@ -0,0 +1,51 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: fireflyiii/data-importer:$$cap_fidi_version
+        environment:
+            FIREFLY_III_ACCESS_TOKEN: $$cap_ff_token
+            FIREFLY_III_URL: $$cap_ff_url
```

Comment:
Do you need `$$cap_ff_url` as a separate variable? Can you not replace it with `$$cap_appname.$$cap_root_domain` ?

Reply:
This is not the data importer URL, but a Firefly III URL (which is a separate app). It can either live on local machine or outside, hence it's needed as a separate variable.